src/app/components/MediaLoader/configs/livePage.ts

```diff
@@ -47,7 +47,9 @@ export default ({
 
   const kind = video?.version?.kind || 'programme';
 
-  const guidanceMessage = video?.guidance?.warnings?.short;
+  const guidanceMessage =
```

Comment:
I think this was my fault 😱 and I typed it wrong. Looks like `video.version.guidance` is correct here, so I'd remove the `video?.guidance?.warnings?.short` check and remove it from the `clipMediaBlock` type as well.